wait-for-it.yaml

```diff
@@ -0,0 +1,27 @@
+package:
+  name: wait-for-it
+  version: 0.1.0
+  epoch: 0
+  description: Pure bash script to test and wait on the availability of a TCP host and port
+  target-architecture:
+    - all
+  copyright:
+    - paths:
+        - "*"
+      attestation: TODO
+      license: MIT
+environment:
+  contents:
```

Comment:
@marcofranssen, do you think that we should include other packages to make wait-for-it work?